buildsystem/templates/Doxyfile.in

```diff
@@ -916,8 +916,8 @@ HTML_STYLESHEET        =
 # since it does not replace the standard style sheet and is therefor more
 # robust against future updates. Doxygen will copy the style sheet file to
 # the output directory.
-
-HTML_EXTRA_STYLESHEET  =
+#ASDF
+HTML_EXTRA_STYLESHEET  = "doxygen_custom.css"
```

Comment:
here, you can add a variable replacement so the css file is found. (`@CMAKE_SOURCE_DIR@/buildsystem/templates/doxygen_custom.css`). Then the configuration of a `.css.in` file is not necessary.